test/e2e/performanceprofile/functests/10_performance_ppc/10_ppc_suite_test.go

```diff
@@ -0,0 +1,58 @@
+package __performance_ppc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	testclient "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/client"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	nto_downstream_image_registry = "registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cluster-node-tuning-operator@sha256"
```

Comment:
no internal references in public code please

Reply:
ack, fixed in the latest commit